openedx/openedx-platform
# feat: update page renders for studio-frontend pages

## Description

This change redirects from Studio to Course Authoring MFE for the Accessibility Accommodation Request page and the Course Checklist page. These pages are being moved to out of studio-frontend and into frontend-app-course-authoring because studio-frontend is being deprecated. This change impacts Authors.

## Supporting information

JIRA Ticket: [TNL-10756](https://2u-internal.atlassian.net/browse/TNL-10756)

> Find all the links for the accessibility page and course checklist and update the reference with the link to course-authoring pages. The links can be found in the list below, but is not an exhaustive list: header, footer, course outline
>
> OUT OF SCOPE: Creating the URLs has been done in a previous sub-task, and is not in scope here.
>
> ACs:
>
> - Ensure there is no impact to the content or functional behavior of the Accessibility and Checklist pages in Studio.
> - Once this work is done, the Accessibility and Checklist pages reference new implementations / can be reached by new URLs.

## Testing instructions

Accessibility Page
1. Using Django Admin enable waffle switch `studio.enable_policy_page`
2. Set `ENABLE_ACCESSIBILITY_PAGE` in `env.development` to true
3. Navigate to any Studio page
4. Scroll to the footer
5. Click "Accessibility Accommodation Request"
6. Should be navigated to the course authoring MFE

Checklist Page
1. Open a course
2. Using the "Tools" dropdown click on Checklist
3. Should be navigated to the course authoring MFE
4. Navigate to the Course Outline
5. Find the Checklist section and click on the link
7. Should be navigated to the course authoring MFE

## Deadline

None

## Other information

This change is dependent on frontend-app-course-authoring PR [#861](https://github.com/openedx/frontend-app-course-authoring/pull/861) and [#870](https://github.com/openedx/frontend-app-course-authoring/pull/870).

This change is part of studio-frontend [DEPR-381](https://github.com/openedx/studio-frontend/issues/381).
